tracex_project/extraction/logic/orchestrator.py

```diff
@@ -99,18 +107,24 @@ def get_db_objects_id(self, object_name):
         """Return the database id objects for the orchestrator instance."""
         return self.db_objects_id[object_name]
 
+    def update_modules(self, modules_list):
+        """Update the modules of the orchestrator instance."""
+        modules_dictionary = self.get_configuration().modules
+        updated_modules = {
+            key: modules_dictionary[key]
+            for key in modules_dictionary
```

Comment:
I would suggest to write it in a one-liner as intended to improve the understanding